### DISCUSSION TITLE
Migrate milvus data with copy ./db directory with Mysql

### DISCUSSION THREAD

**Discussion by loklok-infi** [milvus-io/milvus#4607] [Q&A and General discussion] [answered]
I use Milvus with MySQL and want to migrate data between Milvus servers, I checked the FAQ, it said I could just copy the db directory to another Milvus server, do I need to do something with the metadata in MySQL?

By the way, if I only want to migrate one collection, could I just copy db/<collection_name> to the new Milvus server?

**Comment by shiyu22** [comment, marked as answer]
Which version are you using?

**Reply by loklok-infi** [reply]
0.10.5 for now

**Reply by shiyu22** [reply]
Hi, you are right, it needs to migrate the MySQL data as well. As for one collection, you can copy db/<collection_name> and MySQL info to the new Milvus server.

Please take care to shut down Milvus during the migration process. BTW, we are developing milvusdm(Milvus Data Migration) tools for you guys to migrate data, so stay tuned.

**Reply by loklok-infi** [reply]
Thanks, really need a milvusdm to handle this
